pallets/liquidity-pools/src/message.rs

```diff
@@ -17,16 +17,16 @@ pub const TOKEN_NAME_SIZE: usize = 128;
 // The fixed size for the array representing a tranche token symbol
 pub const TOKEN_SYMBOL_SIZE: usize = 32;
 
-/// A Connector Message
+/// A LiquidityPools Message
 ///
-/// A connector message requires a custom decoding & encoding, meeting the
-/// Connector Generic Message Passing Format (CGMPF): Every message is encoded
-/// with a u8 at head flagging the message type, followed by its field. Integers
-/// are big-endian encoded and enum values (such as `[crate::Domain]`) also have
-/// a custom CGMPF implementation, aiming for a fixed-size encoded
+/// A message requires a custom decoding & encoding, meeting the
+/// LiquidityPool Generic Message Passing Format (GMPF): Every message is
```

Comment:
So we decided on renaming CGMPF to GMPF?

Reply:
I mean, it's not a name it's just an abbreviation. With "Connectors" we abbreviated it to `[C]onnectors[G]eneral[M]essage[P]assing[F]ormat` but for the sake of avodiding crazy and ad-hoc abbreviations I didn't make it `LQGMPF` but just used the general `GMPF` here. Hope that makes sense!

Reply:
I fully agree. The main reason for the comment was to raise awareness to other potential reviewers. I know that these huge renaming refactors sometimes make unwanted changes which can be easily overlooked.

Reply:
I see :) thanks!